### DISCUSSION TITLE
Title text for fluid field buttons?

### DISCUSSION THREAD

**Discussion by murtaugh** [ExpressionEngine/ExpressionEngine#2945] [Feature Requests]
Fluid fields can be complex, and teaching people (or remembering) how to use them can be a challenge.

It would help with user education if we could add text to each button that would be applied as a `title` attribute of each button's `a` tag.

**Comment by ops-andy** [comment]
@murtaugh specifically which buttons are you thinking need  a title tag? Can you share some screenshots or code snippet to show us what you have in mind? thanks

**Comment by murtaugh** [comment]
I mean the `title` attribute of the `a` tag, so we can use the standard browser method for showing a tooltip on hover. (Screenshot of an active tooltip on a fluid field button attached.) It would then also be available for enhancement via JS.

Suggested markup:

```
<a href="#" 
	class="button button--auto button--default button--small" 
	data-field-name="prose_fluid_card_external"
	title="insert a content card that links to an outside website">
	<img src="/?ACT=27&amp;addon=grid&amp;file=icon.svg" width="16" height="16" alt=" Related Content (external)"><br>
	Related Content (external)
</a>
```

<img width="536" alt="Screen Shot 2021-04-20 at 10 46 44 AM" src="https://user-images.githubusercontent.com/308752/115416351-b5767280-a1c5-11eb-9d9a-2c77baebda26.png">

(The icon image shouldn't have alt text that replicates the text in the link, but I'll submit that separately.)

**Comment by intoeetive** [comment]
Just noting that if / when we'll implement this, it needs to be done together with https://github.com/ExpressionEngine/ExpressionEngine/issues/2350 / https://github.com/ExpressionEngine/ExpressionEngine/issues/619
